src/helpers/constants/error.messages.js

```diff
@@ -5,5 +5,7 @@ export default {
   emptyBody: 'Server unable to process the recieved data',
   unkownEmail: "A user with the provided email doesn't exist",
   invalidLink: 'Invalid link provided',
-  incorectPassword: 'Incorect password provided'
+  incorectPassword: 'Incorect password provided',
+  emailLinkInvalid: 'The link provided is  corrupt, please request a new one or try to click it again',
+  noUser: 'User doesn\'t exist.'
```

Comment:
please, @truestbyheart let's be consistent and use double quotes whenever a string contains an escape character.